### ISSUE TITLE
Text overlapping

### ISSUE THREAD

**Issue by atrailin** [GoodieBag/CarouselPicker#6]
Hello 
In text and mixed  modes  all list items overlap each other badly. My words are 5-8 characters long. I've tried different font sizes 
Is there a way to remove or reduce overlap?

**Comment by atrailin** [comment]
I've managed to work around this issue by leaving every second list item blank ( or filled with few spaces)

**Comment by Vatican-Cameos** [comment]
Yes i think the overlap issue might come up when the character length is long. I will try to dig around this issue and try to find a solution. But glad you could come up with a work around for now. Sorry for the inconvenience.

**Comment by augustolecona** [comment]
Hi Vatican-Cameos, I think your framework it's very easy to use and I was wondering if you have managed to fix the overlapping bug. Thanks in advance
